app/assets/javascripts/organizations.js

```diff
@@ -10,9 +10,15 @@ $(document).ready(function() {
   if (('.org-address').length >= 1 ) {
     displayMap($('.org-address').text());
   }
+
+  //not set to display none so that the page doesn't jump up
+  $('.flash-notice').delay(1000).animate({ opacity: 0 });
```

Comment:
There are four types of flashes: `alert`, `error`, `notice`, and `success`. Should we expand this to handle each type of flash? We'd run the risk of the user missing important error messages, but it could still be a better interaction than leaving it on the screen. What do you think?

If you want to go that route, you might want to target the `div.flashes` that wraps all of the individual flash divs.